numerical_methods/false_position.cpp

```diff
@@ -19,56 +23,87 @@
 #include <iostream>
 #include <limits>
```

Comment:
Add a brief description of the headers (what do they add/do).
```suggestion
#include <iostream>   /// for io operations
#include <limits>     ///
```